File: Lib/Telerik/Examples/QuickStart/GridView/Performance/HighRefresh/IMockDataSource.vb
Imports System.Text

Namespace Telerik.Examples.WinControls.GridView.Performance.HighRefresh
	Friend Interface IMockDataSource(Of T)
		ReadOnly Property Source() As List(Of T)
		Sub Refresh()
		ReadOnly Property Rows() As Integer
		ReadOnly Property Columns() As Integer
	End Interface
End Namespace

File: Lib/Telerik/Examples/QuickStart/GridView/Performance/HighRefresh/IMockDataSource.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Telerik.Examples.WinControls.GridView.Performance.HighRefresh
{
    interface IMockDataSource<T>
    {
        List<T> Source { get;}
        void Refresh();
        int Rows { get;}
        int Columns { get;}
    }
}
